src/control/server/config.go

```diff
@@ -369,20 +369,12 @@ func (c *Configuration) SetNvmeShmID(base string) {
 }
 
 // SetPath sets the default path to the configuration file.
-func (c *Configuration) SetPath(path string) error {
+func (c *Configuration) SetPath(path string) (err error) {
 	if path != "" {
-		c.Path = path
+		c.Path, err = c.ext.resolvePath(path)
```

Comment:
It's a minor quibble, but if resolvePath() fails then c.Path will be set to an empty string. Before, if the path resolution failed then the original path would be preserved. In practice, will this matter? Dunno. Wanted to point out the change in behavior, though.